qcodes/utils/dataset/doNd.py

```diff
@@ -101,7 +109,21 @@ def do0d(
         The QCoDeS dataset.
     """
     meas = Measurement()
-    _register_parameters(meas, param_meas)
+
+    measured_parameters = tuple(param for param in param_meas
+                                if isinstance(param, _BaseParameter))
+
+    try:
+        shapes: Shapes = detect_shape_of_measurement(
+            measured_parameters,
+        )
+    except TypeError:
+        LOG.exception(
+            f"Could not detect shape of {measured_parameters} "
```

Comment:
should we add more info about the measured parameters into the message or will the exception's stack trace give us enough?

Reply:
I think the exception should be enough for now